Tasks/Common/artifacts-common/stringUtils.ts

```diff
@@ -0,0 +1,21 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+
+export function getProjectAndFeedIdFromInputParam(inputParam: string): any {
+    const feedProject = tl.getInput(inputParam);
+    return getProjectAndFeedIdFromInput(feedProject);
+}
+
+export function getProjectAndFeedIdFromInput(feedProject: string): any {
```

Comment:
Create an actual type for { feedId:string, projectId: string}? And possibly rename function based on that (or at least remove "FromInput" which doesn't really convey anything)?